dragonfly/world/mcdb/provider.go

```diff
@@ -117,12 +118,7 @@ func (p *Provider) SetWorldName(name string) {
 
 // WorldSpawn returns the spawn of the world as present in the level.dat.
 func (p *Provider) WorldSpawn() world.BlockPos {
-	y := p.d.SpawnY
-	if p.d.SpawnY > 256 {
```

Comment:
Vanilla will have the SpawnY set to the maximum value by default and finds a safe spawn at the highest block. Removing this will spawn the player miles in the sky.

Reply:
Ah I see, that makes a lot more sense now. Would keeping the world spawn function how it is but adding a check to the initial load world sequence to see if the spawn Y is above 256, and if so, setting the spawn to the highest block be a good idea?

Reply:
Yeah that sounds fair enough.

Reply:
Alright, should be done then